### ISSUE TITLE
SFTP not working after update

### ISSUE THREAD

**Issue by diegojsRW** [liximomo/vscode-sftp#260]
## Expected Behavior
- SFTP showing on status bar
- SFTP commands on the file explorer's context menu
- SFTP upload on save

## Actual Behavior
- No SFTP on status bar
- No SFTP command on the context menu
- SFTP doesn't upload on save
- The only command present from SFTP, "SFTP: Config", yields an error "command 'sftp.config.default' not found" 

## Steps to Reproduce the Problem

  1. Update the SFTP extension
  1. Open an workspace previously configured with SFTP

### Extension Logs
```
[Extension Host] Activating extension `liximomo.sftp` failed:  Cannot find module './modules/AppState'
[Extension Host] Here is the error stack:  Error: Cannot find module './modules/AppState'
	at Function.Module._resolveFilename (module.js:470:15)
	at Function.Module._load (module.js:418:25)
	at Function.t._load (/opt/visual-studio-code/resources/app/out/vs/workbench/node/extensionHostProcess.js:814:682)
	at Function.o._load (/opt/visual-studio-code/resources/app/out/vs/workbench/node/extensionHostProcess.js:765:373)
	at Module.require (module.js:498:17)
	at Module.patchedRequire [as require] (/opt/visual-studio-code/resources/app/extensions/git/node_modules/diagnostic-channel/dist/src/patchRequire.js:14:46)
	at require (internal/module.js:20:19)
	at Object.<anonymous> (/home/diego/.vscode/extensions/liximomo.sftp-1.2.7/out/src/app.js:6:20)
	at Object.<anonymous> (/home/diego/.vscode/extensions/liximomo.sftp-1.2.7/out/src/app.js:20:3)
	at Module._compile (module.js:571:32)
	at Object.Module._extensions..js (module.js:580:10)
	at Module.load (module.js:488:32)
	at tryModuleLoad (module.js:447:12)
	at Function.Module._load (module.js:439:3)
	at Function.t._load (/opt/visual-studio-code/resources/app/out/vs/workbench/node/extensionHostProcess.js:814:682)
	at Function.o._load (/opt/visual-studio-code/resources/app/out/vs/workbench/node/extensionHostProcess.js:765:373)
	at Module.require (module.js:498:17)
	at Module.patchedRequire [as require] (/opt/visual-studio-code/resources/app/extensions/git/node_modules/diagnostic-channel/dist/src/patchRequire.js:14:46)
	at require (internal/module.js:20:19)
	at Object.<anonymous> (/home/diego/.vscode/extensions/liximomo.sftp-1.2.7/out/src/extension.js:3:15)
	at Object.<anonymous> (/home/diego/.vscode/extensions/liximomo.sftp-1.2.7/out/src/extension.js:45:3)
	at Module._compile (module.js:571:32)
	at Object.Module._extensions..js (module.js:580:10)
	at Module.load (module.js:488:32)
	at tryModuleLoad (module.js:447:12)
	at Function.Module._load (module.js:439:3)
	at Function.t._load (/opt/visual-studio-code/resources/app/out/vs/workbench/node/extensionHostProcess.js:814:682)
	at Function.o._load (/opt/visual-studio-code/resources/app/out/vs/workbench/node/extensionHostProcess.js:765:373)
	at Module.require (module.js:498:17)
	at Module.patchedRequire [as require] (/opt/visual-studio-code/resources/app/extensions/git/node_modules/diagnostic-channel/dist/src/patchRequire.js:14:46)
	at require (internal/module.js:20:19)
	at Function.i [as __$__nodeRequire] (/opt/visual-studio-code/resources/app/out/vs/loader.js:30:770)
	at /opt/visual-studio-code/resources/app/out/vs/workbench/node/extensionHostProcess.js:790:186
	at t._doActivateExtension (/opt/visual-studio-code/resources/app/out/vs/workbench/node/extensionHostProcess.js:790:298)
	at t._activateExtension (/opt/visual-studio-code/resources/app/out/vs/workbench/node/extensionHostProcess.js:789:187)
	at Object.actualActivateExtension (/opt/visual-studio-code/resources/app/out/vs/workbench/node/extensionHostProcess.js:787:48)
	at e._activateExtension (/opt/visual-studio-code/resources/app/out/vs/workbench/node/extensionHostProcess.js:452:188)
	at /opt/visual-studio-code/resources/app/out/vs/workbench/node/extensionHostProcess.js:451:757
	at Array.map (native)
	at e._activateExtensions (/opt/visual-studio-code/resources/app/out/vs/workbench/node/extensionHostProcess.js:451:732)
	at e.activateById (/opt/visual-studio-code/resources/app/out/vs/workbench/node/extensionHostProcess.js:450:247)
	at t.activateById (/opt/visual-studio-code/resources/app/out/vs/workbench/node/extensionHostProcess.js:787:610)
	at t.<anonymous> (/opt/visual-studio-code/resources/app/out/vs/workbench/node/extensionHostProcess.js:812:334)
	at /opt/visual-studio-code/resources/app/out/vs/workbench/node/extensionHostProcess.js:88:680
	at Object.next (/opt/visual-studio-code/resources/app/out/vs/workbench/node/extensionHostProcess.js:88:785)
	at s (/opt/visual-studio-code/resources/app/out/vs/workbench/node/extensionHostProcess.js:87:782)
	at Object.g [as _notify] (/opt/visual-studio-code/resources/app/out/vs/workbench/node/extensionHostProcess.js:77:729)
	at Object.enter (/opt/visual-studio-code/resources/app/out/vs/workbench/node/extensionHostProcess.js:81:61)
	at n.Class.derive._oncancel._run (/opt/visual-studio-code/resources/app/out/vs/workbench/node/extensionHostProcess.js:82:866)
	at n.Class.derive._oncancel._completed (/opt/visual-studio-code/resources/app/out/vs/workbench/node/extensionHostProcess.js:82:307)
	at Object.g [as _notify] (/opt/visual-studio-code/resources/app/out/vs/workbench/node/extensionHostProcess.js:77:729)
	at Object.enter (/opt/visual-studio-code/resources/app/out/vs/workbench/node/extensionHostProcess.js:81:61)
	at n.Class.derive._oncancel._run (/opt/visual-studio-code/resources/app/out/vs/workbench/node/extensionHostProcess.js:82:866)
	at n.Class.derive._oncancel._completed (/opt/visual-studio-code/resources/app/out/vs/workbench/node/extensionHostProcess.js:82:307)
	at FSReqWrap.cb [as oncomplete] (fs.js:260:19)
``` 

## Specifications

  - Version: 
```
Visual Studio Code
Version: 1.25.1
Commit: 1dfc5e557209371715f655691b1235b6b26a06be
Date: 2018-07-11T15:40:20.190Z
Electron: 1.7.12
Chrome: 58.0.3029.110
Node.js: 7.9.0
V8: 5.8.283.38
```

Architecture: x64
  - Platform: Arch Linux x86_64 4.17.5-1-ARCH Cinnamon 3.8.8

**Comment by diegojsRW** [comment]
I solved the problem renaming $HOME/.vscode/extensions/liximomo.sftp-1.2.7/out/src/modules/appState.js to AppState.js.
The problem was the case-sensitive filesystem naming.

**Comment by liximomo** [comment]
I've fixed it in v1.2.8.
